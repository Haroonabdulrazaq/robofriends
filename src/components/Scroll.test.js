import react from 'react';
import { shallow } from 'enzyme';
import Scroll from './Scroll';
import setup from '../setupTest';

 

it('should the Scroll component match snapShot', ()=>{
    expect(shallow(<Scroll />)).toMatchSnapshot()
})
