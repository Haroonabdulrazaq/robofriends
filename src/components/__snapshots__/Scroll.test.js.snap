// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should the Scroll component match snapShot 1`] = `ShallowWrapper {}`;
